modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage/corrupted_tongue.dm

```diff
@@ -118,13 +117,15 @@
 	// if you drained from a human with a client, congrats
 	var/drained_multiplier = (is_target_human_with_client ? BLOOD_DRAIN_MULTIPLIER_CKEY : 1)
 
-	victim.blood_volume = clamp(victim.blood_volume - drained_blood, 0, BLOOD_VOLUME_MAXIMUM)
 	hemophage.blood_volume = clamp(hemophage.blood_volume + (drained_blood * drained_multiplier), 0, BLOOD_VOLUME_MAXIMUM)
 
 	log_combat(hemophage, victim, "drained [drained_blood]u of blood from", addition = " (NEW BLOOD VOLUME: [victim.blood_volume] cL)")
 	victim.show_message(span_danger("[hemophage] drains some of your blood!"))
 	to_chat(hemophage, span_notice("You drink some blood from [victim]![is_target_human_with_client ? " That tasted particularly good!" : ""]"))
 
+	var/obj/item/organ/internal/stomach/hemophage/stomach_reference = hemophage.get_organ_slot(ORGAN_SLOT_STOMACH)
+	victim.transfer_blood_to(stomach_reference, drained_blood, forced = TRUE)
```

Comment:
Won't this basically double the amount of blood you get from drinking from someone? Since we already add the drained_blood before.

Reply:
Just tested, we're good.

Reply:
To be safe, I'd move this back up to where the original code to remove the blood was at, just so that in case it runtimes, it doesn't give the Hemophage free blood without taking it away from the person they're sucking. I'd also probably do something in case they don't have a stomach, because as-is I think it's going to runtime without a stomach.